src/components/Page/Page.jsx

```diff
@@ -10,14 +10,37 @@ import Gitter from '../Gitter/Gitter';
 // Load Styling
 import './Page.scss';
 
+// Placeholder string
```

Comment:
move to placeholder component

Reply:
I did that first, but since it just returns a string I thought it would be confusing it's not a React component. I might switch to that.

Reply:
yeah a component would be perfect imo